### PR TITLE
Improved the ACCESS privilege definition

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala [hunk 1]
```diff
@@ -61,8 +61,8 @@ class SecurityAdministrationTest extends DocumentingTest with QueryStatisticsTes
       // image source: https://docs.google.com/drawings/d/1Zc5eawW58r4CI8e4FeYnA8cwUyIJX-7B4Eecj-qkd_8/edit
       section("The database `ACCESS` privilege", "administration-security-administration-database-access", "enterprise-edition") {
         p(
-          """The `ACCESS` privilege can be used to enable the ability to access a database.
-            |If this is not granted to users, they will not even be able to start transactions on the relevant database.""".stripMargin)
+          """The `ACCESS` privilege enables users to read data and perform updates in a database.
```
Comment (craigtaverner) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala]: The access privilege does not provide any read or write privileges, so this sentence is false. It allows connecting to the database only, and any reads or writes will not work (reads will return no results and writes will cause an error).

**Review by craigtaverner — CHANGES_REQUESTED** [review]

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala (hunk 1 above)
Comment (renetapopova) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityAdministrationTest.scala]: Thanks @craigtaverner, I see what you mean. I've misinterpreted the sentence "If this is not granted to users, they will not be able to read any data, or perform any updates to the database."

**Review by craigtaverner — APPROVED** [review]
Much better. I wonder if it would not be a good idea to also mention that it is insufficient to allow reading and writing the database? I think that can be deduced from this content, but sometimes it helps to be explicit.

**Review by davidoliverSP2 — APPROVED** [review]
👍  thanks @renetapopova

**Pull request merged**